modules/nf-core/nail/search/main.nf

```diff
@@ -0,0 +1,58 @@
+process NAIL_SEARCH {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nail:0.3.0--h4349ce8_1':
+        'biocontainers/nail:0.3.0--h4349ce8_1' }"
+
+    input:
+    tuple val(meta), path(query)
+    path target
+    val write_align
+
+    output:
+    tuple val(meta), path("${prefix}.txt"), emit: output
+    tuple val(meta), path("${prefix}.tbl"), emit: target_summary
+    tuple val(meta), path("${prefix}.ali"), emit: alignments, optional: true
+    path "versions.yml"                   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args    = task.ext.args ?: ''
+    prefix      = task.ext.prefix ?: "${meta.id}"
+    alignment   = write_align ? "--ali-out ${prefix}.ali" : ''
```

Comment:
```suggestion
    def alignment = write_align ? "--ali-out ${prefix}.ali" : ''
```
To avoid any weird issues later on since this variable isn't used outside of the `script` block

Reply:
Ah okay that's a known bug in Nextflow :) It should be fixed in the next version. Fine by me to keep it like this for now, but can you add a comment explaining why there is no `def` here?